src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,116 @@
+@import '../../styles/utils/mixins';
+@import '../../styles//utils/vars';
+
+.footer {
+  padding: 32px;
+  margin-top: 80px;
+  box-shadow: 0px -1px 0px $elements;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+
+    margin: auto;
+    max-width: 1130px;
+
+    @include onPhone {
+      display: block;
+    }
+  }
+
+  &__logo {
+    margin-top: 14px;
+    @include onPhone {
+      margin-bottom: 32px;
+    }
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+
+    @include onPhone {
```

Comment:
As I said you can write rules for phone without mixin. And only change this rules for mixins onTablet and onDesktop

Reply:
but the Phone format is very different from the other two, so it's more convenient to use it than to write the rules twice for Tablet and Desktop